packages/@o3r/workspace/schematics/ng-add/index.ts

```diff
@@ -0,0 +1,57 @@
+import { chain, externalSchematic, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import * as fs from 'node:fs';
+import * as path from 'node:path';
+import { lastValueFrom } from 'rxjs';
+import type { PackageJson } from 'type-fest';
+import { AddDevInstall, isPackageInstalled, registerPackageCollectionSchematics } from '@o3r/schematics';
+import { NgAddSchematicsSchema } from './schema';
+import { RepositoryInitializerTask } from '@angular-devkit/schematics/tasks';
+import { prepareProject } from './project-setup/index';
+
+/**
+ * Add Otter library to an Angular Project
+ * @param options
+ */
+export function ngAdd(options: NgAddSchematicsSchema): Rule {
+  return async (tree: Tree, context: SchematicContext) => {
+    const ownPackageJsonContent = JSON.parse(fs.readFileSync(path.resolve(__dirname, '..', '..', 'package.json'), {encoding: 'utf-8'})) as PackageJson;
+    const o3rCoreVersion = ownPackageJsonContent.version ? `@${ownPackageJsonContent.version}` : '';
```

Comment:
I don;t think you need `@o3r/core` check for this package. It is not a dependency of `@o3r/workspace` package.

Reply:
Not sure to clearly understand the comment.
If it is referring to the `o3rCoreVersion` variable name, it's a copy/past, I will change the name to `o3rVersion`.
If it is referring to the check of the installation of `@o3r/core`, It is actually necessary at workspace level to provide the different schematics without have to always specify the `yarn workspace ....` in case of `pnp` setup.